packages/framework-provider-local/src/library/events-adapter.ts

```diff
@@ -88,8 +88,6 @@ export async function storeEvents(
     )
   }
   logger.debug('[EventsAdapter#storeEvents] EventEnvelopes stored')
-
-  await userApp.boosterEventDispatcher(eventEnvelopes)
```

Comment:
I'm not 100% sure if this could be removed but if we call this method a class is needed to handle the dispatcher. Could anyone confirm if it's needed or not?

Reply:
I just restore it as it was needed for the Local Provider due to this is a sync process